### PR TITLE
Addressing bunch of small nits in AppNotifications sample

### PR DESCRIPTION
**Pull request by loneursid**
## Description

Addressing bunch of small nits in AppNotifications sample. Cheap improvements that should be quick to review / approve

## Target Release

WindowsAppSDK v1.1

## Checklist

- [ ] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [ ] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release).
- [ ] Samples set the minimum supported OS version to Windows 10 version 1809.
- [ ] Samples build clean with no warnings or errors.
- [ ] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).
- [ ] If I am onboarding a new feature, then I must have correctly setup a new CI pipeline for my feature with the correct triggers and path filters laid out in the "Onboarding Samples CI Pipeline for new feature" section in samples-guidelines.md.
- [ ] I have commented on my PR `/azp run SamplesCI-<FeatureName>` to have the CI build run on my branch for each of my FeatureName my PR is modifying. This must be done on the latest commit on the PR before merging to ensure the build is up to date and accurate. Warning: the PR will not block automatically if this is not run due to '/azp run' limitation on triggering more than 10 pipelines.

### CODE REVIEW

File: Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/ToastWithTextBox.cpp [hunk 1]
```diff
@@ -44,21 +44,16 @@ bool ToastWithTextBox::SendToast()
             </toast>" };
 
     auto toast{ winrt::AppNotification(xmlPayload) };
-    toast.Priority(winrt::AppNotificationPriority::High);
```
Comment (sharath2727) [inline Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/ToastWithTextBox.cpp:47]: nit: we don't need this?

Comment (danielayala94) [reply Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/ToastWithTextBox.cpp:47]: Most app notifications don't need this. High priority is only useful for a very specific Settings thing.

**Review by sharath2727 — APPROVED** [review]

**Review by pmpurifoy — APPROVED** [review]

File: Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/App.xaml.cpp [hunk 2]
```diff
@@ -51,7 +51,7 @@ namespace winrt::CppUnpackagedAppNotifications::implementation
 
         HWND hwnd;
         auto WindowNative{ app->window.as<IWindowNative>() };
-        if (WindowNative != nullptr && WindowNative->get_WindowHandle(&hwnd) == S_OK)
+        if (WindowNative != nullptr && SUCCEEDED(WindowNative->get_WindowHandle(&hwnd)))
```
[the remark this answers is not in the document]
Comment (loneursid) [reply Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/App.xaml.cpp]: done

File: Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/Common.h [hunk 3]
```diff
@@ -8,6 +8,6 @@ struct Common
     static std::optional<std::wstring> ExtractParamFromArgs(std::wstring const& args, std::wstring const& paramName);
 
     static std::wstring MakeScenarioIdToken(unsigned Id);
-    static std::optional <unsigned> ExtractScenarioIdFromArgs(std::wstring const& string);
+    static std::optional<unsigned> ExtractScenarioIdFromArgs(std::wstring const& string);
```
[the remark this answers is not in the document]
Comment (loneursid) [reply Samples/Notifications/App/CppUnpackagedAppNotifications/CppUnpackagedAppNotifications/Notifications/Common.h]: done

**Pull request merged**